tslint.json

```diff
@@ -13,24 +20,38 @@
   "rules": {
     "quotemark": false,
     "trailing-comma": false,
-    "curly": ["true", "same-line-single-line"],
+    "curly": [
+      "true",
+      "same-line-single-line"
+    ],
     "object-literal-sort-keys": false,
     "arrow-return-shorthand": true,
     "prefer-method-signature": true,
     "arrow-parens": false,
-    "no-unnecessary-type-assertion": true,
-    "array-type": [true, "array"],
+    "no-unnecessary-type-assertion": false,
```

Comment:
?

Reply:
Was tossing errors. Im like 80% sure this is only necessary because i didn't do this https://github.com/MyCryptoHQ/MyCrypto/pull/3441#discussion_r472984320 . Will try it and see what happens